src/password/Confirm.vue

```diff
@@ -7,9 +7,24 @@
 
       <v-form @submit.prevent="confirm" ref="form" class="pa-4">
         <p>{{ $vuetify.lang.t('$vuetify.password.confirm.explanation') }}</p>
-        
-        <BaseTextField type="password" :label="$vuetify.lang.t('$vuetify.password.confirm.pwInput')" v-model="password" 
-                       :rules="rules" :error-messages="errors" validate-on-blur @keyup.enter="blur" autofocus name="password" />
+
+        <div class="password">
+          <BaseTextField v-if="passwordIsHidden" type="password" :label="$vuetify.lang.t('$vuetify.password.confirm.pwInput')" v-model="password" 
+          :rules="rules" :error-messages="errors" validate-on-blur @keyup.enter="blur" autofocus name="password" />
+          
+          <BaseTextField v-else type="text" :label="$vuetify.lang.t('$vuetify.password.confirm.pwInput')" v-model="password" 
+          :rules="rules" :error-messages="errors" validate-on-blur @keyup.enter="blur" autofocus name="password" />
```

Comment:
Is it possible / preferable to just change the `type` attribute of the BaseTextField when the user clicks the show/hide password button, rather than rendering a completely different BaseTextField? I don't know if it matters, but that might help us see more clearly in the code that that's the only difference between the two.

Reply:
Here is an example using Vue 3, so might not be completely the same: https://simedia.tech/blog/show-hide-password-input-values-with-vue-js/

Reply:
Thanks, I had forgotten you could do this in Vue